build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta7'
```

Comment:
I modified that line in order to gradle work with the Stable Android studio (2.3.3) That makes reference to he new one.

Reply:
great!

Reply:
yah next time I make sure dont check in this file

Reply:
Perfect :)